extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
-      </dd>
-      <dt id="stack-components-elasticsearch-clients">
-        Elasticsearch clients
-      </dt>
-      <dd>
-        The clients provide a convenient mechanism to manage API requests and responses to and from Elasticsearch from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. <a href="en/elasticsearch/client/index.html">Learn more about the Elasticsearch clients</a>.
-      </dd>
-    </dl>
-  </details>
-
-  <h2>Get started</h2>
-  <p>New to Elastic? Learn how you can make the most of your data with the Elastic Stack.
-    Get hands-on with a solution and quickly see data in action, or start from a blank page. </p>
-  <div class="row my-4">
-    <div class="col-md-4 col-12 mb-2">
-      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-        <div class="card h-100">
-          <h4 class="mt-3">
-            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
-            Search
-          </h4>
-          <p>Build custom applications with your data using Elasticsearch.</p>
-        </div>
-      </a>
+  <div role="tabpanel" id="solution-tab-contents" aria-labelledby="solution-tab" tabindex="0">
+    <div class="row">
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/elasticsearch/reference/current/index.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
+              Search
+            </h3>
+            <p>Build custom applications with your data using Elasticsearch.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/observability/current/index.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltaa08b370a00bbecc/634d9da14e565f1cdce27f7c/observability-logo-color-32px.png');">
+              </span>
+              Observability
+            </h3>
+            <p>Monitor hosts by collecting logs and metrics with Elastic Observability.</p>
```

Comment:
We have many definitions of observability and none are great. So for now, to sync up with what Joe is doing, let's change this to match serverless...because Observability goes way beyond logs and metrics these days!
```suggestion
            <p>Monitor applications and systems with Elastic Observability.</p>
```

Reply:
I just noticed that we have solution "cards" on the bottom of the Kibana docs page, too. Whatever we use, we should be consistent. https://docs_2805.docs-preview.app.elstc.co/guide/en/kibana/current/index.html

![image](https://github.com/elastic/docs/assets/14206422/490af9be-8a75-40b1-b5a0-ca077d4dd268)